h/zis/services/auth.h

```diff
@@ -51,12 +60,14 @@ typedef struct AuthServiceParmList_tag {
 #define ZIS_AUTH_SERVICE_PARMLIST_FC_VERIFY_PASSWORD 0
 #define ZIS_AUTH_SERVICE_PARMLIST_FC_ENTITY_CHECK 1
 #define ZIS_AUTH_SERVICE_PARMLIST_FC_GET_ACCESS 2
+#define ZIS_AUTH_SERVICE_PARMLIST_FC_GENERATE_TOKEN 3
   char userIDNullTerm[ZIS_AUTH_SERVICE_USER_ID_MAX_LENGTH + 1];
   char passwordNullTerm[ZIS_AUTH_SERVICE_PASSWORD_MAX_LENGTH + 1];
   /* up to 8 characters: */
   char classNullTerm[ZIS_AUTH_SERVICE_CLASS_MAX_LENGTH + 1];
   char options;
 #define ZIS_AUTH_SERVICE_PARMLIST_OPTION_GENERATE_IDT 0x80
```

Comment:
Should `ZIS_AUTH_SERVICE_PARMLIST_OPTION_GENERATE_ID` be deprecated and not be checked since there is a separate function code which already means that action? I'd leave a comment telling not to use 0x80 because there is already client code which might set it.

Reply:
added comment with the text that this parameter is deprecated